src/tseda/datastore.py

```diff
@@ -175,6 +206,11 @@ def __panel__(self):
             else:
                 logger.info("No population defined")
         data = self.data[self.columns]
+
+        # TODO: create them indivudually
+        # ranges not possible
+        filters = self.create_filters()
```

Comment:
yes, do we really need filters for all columns? I also don't really understand how to use the filter for the "selected" column.

Reply:
No, Ill update them!